shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2146,6 +2150,15 @@ - (void)setSelectionRects:(NSArray*)rects {
   _activeView.selectionRects = rectsAsRect;
 }
 
+- (void)startLiveTextInput {
+  if (_activeView == nil || !_activeView.isFirstResponder) {
+    return;
+  }
```

Comment:
put these into `if (@available(iOS 15.0, *))` so we only perform these checks above iOS 15

Reply:
Done.